Code/Yahoo.Yui.Compressor.Tests/TestHelpers/BuildEngineStub.cs

```diff
@@ -27,6 +29,7 @@ public void LogWarningEvent(BuildWarningEventArgs e)
 
         public void LogMessageEvent(BuildMessageEventArgs e)
         {
+            Messages.Add(e.Message);
         }

```

Comment:
Since you aren't checking messages any more, you can completely revert the changes to this file.
Try to keep your PR as tight as possible, doing the minimum necessary to adequately meet the requirement.